Bin/Stub.php

```diff
@@ -0,0 +1,197 @@
+                    $verbose = true;
+
+                    break;
+                case 'h':
+                case '?':
+
+                    return $this->usage();
+                case '__ambiguous':
+                    $this->resolveOptionAmbiguity($v);
+
+                    break;
+            }
+        }
+
+        if (null === $output) {
+            return $this->usage();
+        }
+
+        $hoaPath = 'hoa://Library/';
+        $aliases = [];
+        $finder  = new File\Finder();
+        $finder
+            ->in(resolve($hoaPath, true, true))
+            ->name('#\.php$#')
+            ->maxDepth(100)
+            ->files();
+
+        foreach ($finder as $file) {
+            $pathName = $file->getPathName();
+            $raw      = file_get_contents($pathName);
+
+            if (!preg_match('#flexEntity\(\'([^\']+)#', $raw, $class)) {
+                preg_match('#\nclass_alias\(\'([^\']+)#', $raw, $class);
+            }
+
+            if (empty($class)) {
+                continue;
+            }
+
+            $FQCN      = $class[1]; // Fully-Qualified Class Name
+            $alias     = Core\Consistency::getEntityShortestName($FQCN);
+            $className = substr($alias, strrpos($alias, '\\') + 1);
+
+            preg_match(
+                '#((?:(?:abstract|final)\s)?' .
+                'class|interface|trait)\s+' . $className . '\s#',
+                $raw,
+                $keyword
+            );
+
+            $aliases[] = [
+                'FQCN'      => $FQCN,
+                'alias'     => $alias,
+                'keyword'   => $keyword[1],
+                'className' => $className
+            ];
+
+            if (true === $verbose) {
+                echo
+                    $keyword[1] . ': ' .
+                    $FQCN . ' > ' .
+                    $alias . "\n";
+            }
+        }
+
+        if (true === $dryRun) {
+            return;
+        }
+
+        $out = '<?php ' . "\n";
+        foreach ($aliases as $class) {
+            $ns = substr($class['alias'], 0, strrpos($class['alias'], '\\'));
+
+            $out .=
+                'namespace ' . $ns . ' {' . "\n" .
+                $class['keyword'] . ' ' . $class['className'] .
+                ' extends \\' . $class['FQCN'] . ' {}' . "\n" .
+                '}' . "\n";
+        }
+
+        file_put_contents($output, $out);
+
+        return;
+    }
+
+    /**
+     * The command usage.
+     *
+     * @return  void
+     */
+    public function usage()
+    {
+        echo
+            'Usage   : devtools:stub <options>', "\n",
+            'Options :', "\n",
+             $this->makeUsageOptionsList([
+                 'stub'    => 'Path to stub file',
+                 'dry-run' => 'No written operation',
+                 'verbose' => 'Echo all information',
+                 'help'    => 'This help.'
+             ]), "\n";
+
+        return;
+    }
+}
+
+__halt_compiler();
+Resolve class_alias function for the IDE.
```

Comment:
A much specific description ? something like
`Create stub class of flex entities for enable auto completion on IDE`